bundles/framework/myplaces3/service/MyPlacesService.js

```diff
@@ -13,18 +13,11 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.service.MyPlacesServic
      */
 
     function (sandbox) {
-        // list of loaded categories & myplaces
-        this._categoryList = [];
-        this._placesList = {}; // {layerId:[places]}
-
+        this._placesList = {}; // {categoryId:[places]}
         this._sandbox = sandbox;
-        this.defaultCategory = null;
-        // skipLoading is used for published maps (value by init-method param)
-        // it means we shouldn't load any features on start and also when saving
-        this.skipLoading = false;
+        this.log = Oskari.log(this.getQName());
         this.srsName = sandbox.getMap().getSrsName();
-        this.loadedCategories = false; // TODO remove when places is loaded when category is selected
-        this.loadedPlaces = false; // TODO remove when places is loaded when category is selected
+        this.mapmodule = this._sandbox.findRegisteredModuleInstance('MainMapModule');
```

Comment:
Not a big fan of having mapmodule requirement here. Maybe we should refactor the measurement formatting to another file and import it for use here instead. But if this works now we can do that later.

Reply:
Maybe formatMeasurementResult could be moved to mapmodule util because it  just for formatting. And its used also in placeform and buttonhandler. getMeasurementResult depends on mapmodule and openlayers and it's only for populating measurement result to my places tab.